yarn.lock

```diff
@@ -2,170 +2,158 @@
 # yarn lockfile v1
```

Comment:
Ah I see we still have the root lockfile, we can just delete the `near-api-js` one then.